### PR TITLE
User schema update

### PR DESCRIPTION
**Pull request by Shovalic**

### CODE REVIEW

File: server/database/schemas/User.js [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
+    forest: { type: String, require: false },
+    get_update: { type: Boolean, require: false },
+    bio: { type: String, maxlength: 240 },
+    created_at: { type: Date, default: Date.now, immutable: true },
+    updated_at: { type: Date },
+  },
+  { versionKey: false }
+);
 
+if (process.env.NODE_ENV !== "test") {
```
Comment (omers4) [inline server/database/schemas/User.js:34]: What's that?

File: server/database/schemas/User.js [hunk 2]
```diff
@@ -1,28 +1,40 @@
...
+    first_name: { type: String, maxlength: 20 },
+    last_name: { type: String, maxlength: 20 },
+    username_case: { type: String, required: false },
+    phone_number: { type: String, require: false },
+    birth_year: { type: Number, required: false },
+    email_address: { type: String, required: false },
+    password: { type: String, required: false },
+    forest: { type: String, require: false },
+    get_update: { type: Boolean, require: false },
+    bio: { type: String, maxlength: 240 },
```
Comment (omers4) [inline server/database/schemas/User.js]: Let's remove the bio

File: server/database/schemas/User.js [hunk 3]
```diff
@@ -1,28 +1,40 @@
...
-  last_name: { type: String, maxlength: 20 },
-  bio: { type: String, maxlength: 240 },
-  created_at: { type: Date, default: Date.now, immutable: true },
-  updated_at: { type: Date },
-}, { versionKey: false });
-
-if (process.env.NODE_ENV !== 'test') {
-  MongooseAutoIncrementID.initialise('counters');
+const userSchema = new Schema(
+  {
```
Comment (omers4) [inline server/database/schemas/User.js:9]: where is the city field?

File: server/database/schemas/User.js [hunk 4]
```diff
@@ -1,28 +1,40 @@
...
+      immutable: true,
+    },
+    first_name: { type: String, maxlength: 20 },
+    last_name: { type: String, maxlength: 20 },
+    username_case: { type: String, required: false },
+    phone_number: { type: String, require: false },
+    birth_year: { type: Number, required: false },
+    email_address: { type: String, required: false },
+    password: { type: String, required: false },
+    forest: { type: String, require: false },
```
Comment (omers4) [inline server/database/schemas/User.js]: what will be here? the id of the forest? if so, let's change the name to "forest_id"

**Review by omers4 — CHANGES_REQUESTED** [review]

File: server/database/schemas/User.js (hunk 1 above)
Comment (Shovalic) [reply server/database/schemas/User.js:34]: IDK, it was built in.

File: server/database/schemas/User.js (hunk 2 above)
Comment (Shovalic) [reply server/database/schemas/User.js]: ok :)

File: server/database/schemas/User.js (hunk 3 above)
Comment (Shovalic) [reply server/database/schemas/User.js:9]: supposed to be in line 19. 
don't you see that?

File: server/database/schemas/User.js (hunk 4 above)
Comment (Shovalic) [reply server/database/schemas/User.js]: as we want. 
I didn't know if we meant to write the name or the id.
I'll change it accordingly

File: server/database/schemas/User.js (hunk 3 above)
Comment (omers4) [reply server/database/schemas/User.js:9]: Now that you added it, I see :-)

File: server/database/schemas/User.js [hunk 5]
```diff
@@ -1,28 +1,41 @@
...
+    first_name: { type: String, maxlength: 20 },
+    last_name: { type: String, maxlength: 20 },
+    city: { type: String, maxlength: 20, required: true },
+    username_case: { type: String, required: false },
+    phone_number: { type: String, require: true },
+    birth_year: { type: Number, required: false },
+    email_address: { type: String, required: true },
+    password: { type: String, required: true },
+    forest_id: { type: String, require: true },
+    get_update: { type: MONGOBoolean, require: false },
```
Comment (omers4) [inline server/database/schemas/User.js]: this is not working. fix MONGOBoolean

Comment (Shovalic) [reply server/database/schemas/User.js]: yes, it was a mistake. I can't send another commit

**Review by omers4 — APPROVED** [review]

**Pull request merged**